### ISSUE TITLE
zLinux (os390x) and phantomjs

### ISSUE THREAD

**Issue by OwenBrotherwood** [beeman/loopback-angular-admin#152]
A little "heads up", strongloop.com is now an IBM Company, so I deceided to test node.js on IBM's zLinux (os390x) mainframe based Linux.
And then I needed a demo program: what better than angular-admin.

Unfortunatly, phantomjs has a problem I have to google for.
Not sure how phantomjs is helping angular-admin, or what it does, so I have to do some research.

The installation of loopback-angular-admin was not as painless as on 86 architecture. The reasons can be my own knowledge.

"Work in progress"

```

> phantomjs@1.9.19 install /home/linux1/loopback-angular-admin/loopback-angular-                                                                        admin/node_modules/karma-phantomjs-launcher/node_modules/phantomjs
> node install.js

PhantomJS not found on PATH
Unexpected platform or architecture: linux/s390x
It seems there is no binary available for your platform/architecture
Try to install PhantomJS globally

```

[linux1@zrhel6dot7 loopback-angular-admin]$ cat /etc/redhat-release
Red Hat Enterprise Linux Server release 6.7 (Santiago)

[linux1@zrhel6dot7 loopback-angular-admin]$ node --version
v4.2.3

[linux1@zrhel6dot7 loopback-angular-admin]$ uname -a
Linux zrhel6dot7.novalocal 2.6.32-573.el6.s390x #1 SMP Wed Jul 1 18:26:59 EDT 2015 s390x s390x s390x GNU/Linux

---

http://phantomjs.org/
http://phantomjs.org/documentation/
http://phantomjs.org/build.html :: Work In Progress

**Comment by OwenBrotherwood** [comment]
Aaagh: what _does_ one answer? 
Hope phantomjs is really necessary.

```
[linux1@zrhel6dot7 phantomjs]$ ./build.py
----------------------------------------
               WARNING
----------------------------------------

Building PhantomJS from source takes a very long time, anywhere from 30 minutes
to several hours (depending on the machine configuration). It is recommended to
use the premade binary packages on supported operating systems.

For details, please go the the web site: http://phantomjs.org/download.html.

Do you want to continue (Y/n)?
```

**Comment by beeman** [comment]
Hey @OwenBrotherwood ,

Awesome that you're trying this project out on zLinux!

PhantomJS is a so-called headless WebKit browser used for testing the user interface. 

The current project does not have any of these tests so it should not be needed to include it. I'm not sure how much will break if you simply remove the entry from package.json, but it's worth a try :)

**Comment by OwenBrotherwood** [comment]
Hey @beeman 
yep: after googling, I remembered PhantomJS's function: so many things to remember :)
it is now removed as there was some issues to get it built, for which I deceided not to use the time for (backlog).

As a "nohup grunt serverer": on and off link http://zlinux1.nodeonmainframe.org:3000
A quick  installation as per readme.md, and apparently, admin cannot login.

I will make a quick install on another cloud or virtual linux for Intel CPU to make sure I have understood the basics (again). Then I will go back to os390x.

Br Owen
